contents/handbook/people/benefits.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Benefits
+sidebar: Handbook
+showTitle: true
+---
+
+Outside of [pay and equity](handbook/people/compensation), we offer a number of additional benefits to our teamm.
+
+With everyone being distributed across the work, we do our best to provide comparable benefits and perks for everyone, but they might vary slightly by country.
+
+### 401k/pension contributions
+
+In the UK, we use [Royal London](https://www.royallondon.com/) as our pension provider. The employee contributes 5%, PostHog contributes 4%, but you can opt out, in case you don't want to contribute. 
+
+We are currently in the process of setting up a 401k for our US team. [Deel](https://app.letsdeel.com/), our international payroll provider is currently building a pension product for contractors as well. We will aim to provide the same percentages as in the UK.
+
+### Health Insurance
+
+We currently provide health insurance in the US and the UK. 
+
+In the US, our medical insurance is provided via UnitedHealthcare and managed via our payroll provider [Gusto](https://app.gusto.com/). The plan includes medical, dental and vision insurance. 
+
+PostHog pays 75% of the premium of the platinum plan, you will pay the other 25%. The same conditions apply to added dependents. UnitedHealthcare also offers Gold and Silver plans on a lower cost, so you might end up contributing less, depending on the plan you choose. The exact costs depend on factors like age, we can create a quote in Gusto. 
+
+In the UK, we use [Bupa](https://www.bupa.co.uk/) for our private healthcare (£100 excess per policy year) and [Medicash] as our Cashplan for dental and vision (Children are included for free). 
+
+Both of them create tax liability and they will effect your Personal Allowance each tax year, you can opt out at any time with 1 month notice. 
+
+### Mental health
+
+Our world can feel a bit heavy from time to time and we want to make sure everyone gets support, if they need it. 
```

Comment:
I think i understand that you're referring to the entire world but this feels like 'things often suck at Posthog'

Reply:
I just re-phrased that slightly to make it about the world in general (e.g. Covid, don't want to give too much room for politics etc), rather than PostHog.